articles/metadata/lock.md

```diff
@@ -10,11 +10,11 @@ When using Auth0's [Lock](/libraries/lock) library, you can define and update th
 
 ## Defining User Metadata on Signup
 
-For information on adding `user_metadata` on signup, see the section on Lock [Custom Sign Up Fields](/libraries/lock/v10/new-features#custom-sign-up-fields) 
+For information on adding `user_metadata` on signup, see the section on Lock [Custom Sign Up Fields](/libraries/lock/v10/new-features#custom-sign-up-fields)
 
 ## Working with User Metadata
 
-Once you have [implemented the login functionality](/quickstart/spa/vanillajs#3-implement-the-login) for your Lock instance, you can choose to store the newly-created `id_token`. This token is used to retrieve the user's profile from Auth0 or to call APIs.
+Once you have [implemented the login functionality](/quickstart/spa/vanillajs#3-implement-the-login) for your Lock instance, you can choose to store the newly-created `id_token`. This token is used to retrieve the user's profile from Auth0.

```

Comment:
@cabralmartin I removed the last sentence. 

However, the [Reading User Metadata Properties paragraph](https://github.com/auth0/docs/pull/2953/files#diff-4782dfde2f6b51c1920a3160a58b8836R32) uses the `id_token` to invoke `lock.getProfile`. Also, the [Updating Metadata Properties paragraph](https://github.com/auth0/docs/pull/2953/files#diff-4782dfde2f6b51c1920a3160a58b8836R48) mentions this which doesn't look correct: "By including the user's `id_token` in the `Authorization` header, you can make the appropriate `PATCH` call to the [Update a user](/api/management/v2#!/Users/patch_users_by_id) endpoint."

Is this `getUserProfile` released? I can't find any reference in the docs or the [auth0.js repo](https://github.com/auth0/auth0.js).